packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,922 @@
+}
+```
+
+## CAPTCHA API objects
+
+### CAPTCHA object
+
+#### CAPTCHA properties
+
+The CAPTCHA object defines the following properties:
+
+| Property           | Type                           | Description                                                                                                       |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `name`             | String                         | Required. Name of CAPTCHA |
+| `siteKey`          | String                         | Required. Site key issued from CAPTCHA Vendor to render CAPTCHA in a page.|
+| `secretKey`        | String                         | Required. Secret key issued from CAPTCHA Vendor to perform server-side validation for CAPTCHA token.|
+| `type`             | String                         | Required. Type of CAPTCHA. Available Values are `HCAPTCHA` and `RECAPTCHA_V2`.|
+
+#### CAPTCHA example
+
+```json
+{
+  "name": "myHCaptcha",
+  "siteKey": "copy_your_site_key",
+  "secretKey": "copy_your_secret_key",
+  "type": "HCAPTCHA"
+}
+```
+
+### CAPTCHA Response object
+
+#### CAPTCHA Response properties
+
+The CAPTCHA object defines the following properties:
+
+| Property           | Type                           | Description                                                                                                       |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `id`               | String                         | Required. Unique key for the CAPTCHA|
+| `name`             | String                         | Required. Name of the CAPTCHA |
+| `siteKey`          | String                         | Required. Site key issued from CAPTCHA Vendor to render CAPTCHA in a page.|
+| `type`             | `HCAPTCHA`, `RECAPTCHA_V2`     | Required. Type of CAPTCHA.|
+| `_links`           | [Links](#captcha-links-object)         | Link relations for this object|
+#### CAPTCHA example
+
+```json
+{
+  "id": "cap18c2Ey3iR9BLDb0g4",
+  "name": "myReCaptcha",
+  "siteKey": "your_site_key",
+  "type": "RECAPTCHA_V2",
+  "_links": {
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+
+### CAPTCHA Links object
+
+Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations. The Links object is read-only.
+
+## Org-wide CAPTCHA settings operations
+
```

Comment:
Let's add: 

The org-wide CAPTCHA settings are used to configure which parts of the authentication flow will require users to pass the CAPTCHA logic.